test/unit/bindings.test.ts

```diff
@@ -97,6 +97,24 @@ describe('MongoCryptConstructor', () => {
     });
   });
 
+  describe('options.keyExpirationMS', () => {
+    context('when the number is positive', () => {
+      it('does not error', () => {
+        expect(
+          new MongoCrypt({ kmsProviders: serialize({ aws: {} }), keyExpirationMS: 1000000 })
+        ).to.be.instanceOf(MongoCrypt);
+      });
+    });
+
+    context('when the number is negative', () => {
+      it('throws an error', () => {
+        expect(() => {
+          new MongoCrypt({ kmsProviders: serialize({ aws: {} }), keyExpirationMS: -1000000 });
+        }).to.throw(TypeError);
```

Comment:
Just related to my comment in src - can we assert on the error message thrown?

Reply:
Asserts on the error message now.